src/plugins/1/essentials/strategies/index.md

```diff
@@ -0,0 +1,370 @@
+
+---
+
+## Managing credentials
+
+There are two ways of interfacing credentials management:
+
+* statically, by exposing a `strategies` object
+* dynamically, by using the dedicated [strategy accessors]({{ site_base_path }}plugins/1/accessors/strategies)
+
+Whether strategies are added statically or dynamically, the same object structure is expected:
+
+* `config`: an object containing the strategy configuration
+  * `authenticator`: One of the exposed [authenticators]({{ site_base_path }}plugins/1/essentials/strategies/#registering-authentication-strategies-default) name
+  * `constructor`: {{{deprecated "1.4.0"}}} (use the `authenticator` property instead) The constructor of the Passport.js strategy. Does not support [dynamic strategy registration]({{ site_base_path }}plugins/1/accessors/strategies)
+* `methods`: an object containing the list of exposed methods
+  * `create`: the name of the exposed [`create` function]({{ site_base_path }}plugins/1/essentials/strategies/#create-default)
+  * `delete`: the name of the exposed [`delete` function]({{ site_base_path }}plugins/1/essentials/strategies/#delete-default)
+  * `exists`: the name of the exposed [`exists` function]({{ site_base_path }}plugins/1/essentials/strategies/#exists-default)
+  * `update`: the name of the exposed [`update` function]({{ site_base_path }}plugins/1/essentials/strategies/#update-default)
+  * `validate`: the name of the exposed [`validate` function]({{ site_base_path }}plugins/1/essentials/strategies/#update-default)
+  * `verify`: the name of the exposed [`verify` function]({{ site_base_path }}plugins/1/essentials/strategies/#verify-default)
+  * (optional) `afterRegister`: the name of the exposed [`afterRegister` function]({{ site_base_path }}plugins/1/essentials/strategies/#optional-afterregister-default)
+  * (optional) `getById`: the name of the exposed [`getById` function]({{ site_base_path }}plugins/1/essentials/strategies/#optional-getbyid-default)
+  * (optional) `getInfo`: the name of the exposed [`getInfo` function]({{ site_base_path }}plugins/1/essentials/strategies/#optional-getinfo-default)
+
+Even though each strategy must declare its own set of properties, the same strategy method can be used by multiple strategies.
+
+
+### Optional config properties
+
+The `config` object may contain the following optional properties:
+
+  * `authenticateOptions`: additional options to be provided to the Passport's [authenticate method](http://passportjs.org/docs/authenticate).
+  * `fields`: an array of field names accepted by the plugin to validate credentials. The list is informative only, meant to be used by the [getAllCredentialFields]({{ site_base_path }}api/1/controller-security/get-all-credential-fields/) and the [getCredentialFields]({{ site_base_path }}api/1/controller-security/get-credential-fields) API methods.
+  * `strategyOptions`: options to be provided to the Passport.js strategy constructor
+
+---
+
+## create
```

Comment:
For this method and the subsequent one, maybe you can add the JS doc instead of specifying arguments types in the sentences